elements/lisk-chain/package.json

```diff
@@ -44,8 +44,7 @@
 		"@liskhq/lisk-validator": "0.5.0-alpha.0",
 		"@types/node": "12.12.11",
 		"debug": "4.1.1",
-		"lodash.clonedeep": "4.5.0",
-		"lodash.isequal": "4.5.0"
+		"lodash.clonedeep": "4.5.0"
```

Comment:
Why not to remove this package?

Reply:
Since clonedeep will remain as dependency anyway, and there is another issue to replace with `lisk-utils` in general with #5566